app/components/documentation-link/template.hbs

```diff
@@ -1,3 +1,3 @@
-<a href="https://github.com/transitland/transitland-docs/blob/master/{{linkPath}}" target="_blank">
+<a href="https://transit.land/documentation/{{linkPath}}" target="_blank">
```

Comment:
@rmglennon no need to include the protocol and host. Probably simplest to just use `/documentation/{{linkPath}}` and then it will click through in both dev and prod environments.